@ornikar/eslint-config-react/tests/no-unescaped-entities.js

```diff
@@ -0,0 +1,15 @@
+function App() {
+  return null;
+}
+
+export function NoUnescapedEntities() {
+  return (
+    <App>
+      {/* eslint-disable-next-line react/no-unescaped-entities */}
+      <p>A paragraph with 'quote' is incorrect</p>
```

Comment:
rajouter la version avec ' qui est correcte

Reply:
`<p>A paragraph with {'quote'} is incorrect</p>` est aussi incorrect à cause de jsx-curly-brace-presence donc la je vois pas ce qui est correct

Reply:
```<p>A paragraph with {"'quote'"} is incorrect</p>

Reply:
![Capture d’écran 2022-09-01 à 14 29 14](https://user-images.githubusercontent.com/24406360/187913843-2b7ed8c2-290d-4fc2-a4f9-8d0cd2ffb36e.png)

Reply:
d'ailleurs c'est plutot:
```<p>{"A paragraph with 'quote' is correct"}</p>```

Reply:
Même erreur avec cette ligne !

Reply:
dans ce cas, il faut fixer pour que 

```
<p>A paragraph with 'quote' is incorrect</p>
```
fonctionne

Reply:
(et avoir la version 
```
<p>{"A paragraph with 'quote' is incorrect"}</p>
```

Reply:
Mais du coup dans le fichier de test de no_escaped_entites, on aura un disable-next-line sur une autre règle que le sujet du fichier, bon pour toi ?
![Capture d’écran 2022-09-01 à 14 31 54](https://user-images.githubusercontent.com/24406360/187914360-15b5e993-8826-45f3-9737-7846bdac394d.png)

Reply:
oui

Reply:
ajoute bien tous les cas correctes et incorectes avec tous les symboles qui il y a dans la regle par défaut (pour bien avoir les erreurs si on retente de mettre la régle par default)

Reply:
Au final, les 2 cas restants dans la config par défaut sont déjà override par une autre erreur eslint/ts qui prend la main dessus (et je ne peux donc pas contrôler les alternatives affiché en passant par no-unescaped-entities)
![Capture d’écran 2022-09-01 à 14 41 04](https://user-images.githubusercontent.com/24406360/187916148-819747ef-878c-455c-8328-319a07f17b30.png)

Reply:
ça viendrait de @typescript-eslint/parser pour >

Reply:
oui logique, c'est pas du jsx valide

Reply:
tu peux quand meme mettre les cas en version correcte

Reply:
Je viens de push avec d'autres cas correct et une documentation dans les rules, bon pour toi ?